src/adr/articles/0002-verifiable-credentials.md

```diff
@@ -0,0 +1,112 @@
+---
+layout: layouts/fullwidth.njk
+title: "2. Verifiable Credentials"
+weight: 1
+date: 2021-07-02
+date_view: "2021-07-02"
+review_in: 3 months
+tags: adr
+areas_of_coverage: "Verifiable Credentials"
+status: proposed
+contributors: "John Nolan"
+---
+# {{title}}
+
+## Overview
+
+**Date**: {{date_view}}
+
+### Status
+
+<strong class="govuk-tag">{{status}}</strong>
+
+### Contributors
+
+{{contributors}}
+
+### Areas of coverage
+
+{{areas_of_coverage}}
+
+## Context
+
+We want to give the Citizen a way of using a verifiable digital representation of a LPA. This will enable them to use this to prove they are who they say they are.
+
+If an LPA is no longer a paper deed, then this is one solution to what a digital version could be.
+
+This could be an optional addition to the existing Use a LPA service to allow for a wider demographic of users.
+
+It needs to be fully interopable with
+
+1. Future technology such as mobile phone digital wallets
+2. Industry standard validation and authenticity implementations
+3. Potential for other agencies to use as a proof of identity
+
+We would need to be able to reliably sign the credential for others to validate the authenticity.
+
+We would need the Verifiable Credential to be able to store the appropriate required information for the consumer to understand its responsibilities.
+
+## Technical
+
+### Interoperability - How does this enable the exchange of information
+
+The Verifiable Credentials data model is a [W3C Open Standard](https://www.w3.org/TR/vc-data-model/). As long as the standards are adhered to, interopability will be high.
+
+The signing of a Verifiable Credential should also be standards driven and supported by the wider industry.
+
+### Developer Knowledge - How well known is this in our current skill sets
+
+**Overall**: 2/10
+The data model is well defined and an understood pattern for representing data.
+
+The validity and authenticity aspect of the specification are early in discovery.
+
+If signing was done via a certificate provider setting, such is done for SSL Certification, then the understanding will be higher. It will come with extra challenges on management of the signing keys and the key life cycle.
+
+### Support/Open Source - Is it well supported
+
+The Verifiable Credentials data model is a [W3C Open Standard](https://www.w3.org/TR/vc-data-model/).
```

Comment:
Maybe should note that a few of the specs it builds on are a lot more drafty, e.g.:
- https://w3c-ccg.github.io/ld-proofs/
- https://json-ld.github.io/rdf-dataset-canonicalization/spec/